.github/pull_request_template.md

```diff
@@ -1,11 +1,19 @@
-<!-- Please follow the template -->
-Thanks for contributing! 🍄
+<!--
 
-1. LINKED ISSUES:
-   Does this PR close/fix an existing issue? Write something like `Closes #10`
+Thanks for contributing! 🍄 Do not ignore this template, plz.
 
-2. TEST URLS:
-   Add some test URLs
+1. Does this PR close/fix an existing issue? Write something like `Closes #10`
+2. What pages does this PR affect? Include some URLs where you tested the code or where reviewers can test it.
+3. Add a screenshot if your PR makes visual changes
```

Comment:
Maybe include a few guidelines to make better screenshots? e.g.:
* Clean up the area around the targeted element
* Zoom on the page for small UI changes
* Provide some context only if needed
* Use the "Before" and "After" arrows

I could use those remainders myself :)

Reply:
Yeah I was thinking about guidelines for screenshots in the readme but those either go in the readme (we have guidelines for the description in there) or in contributing

This screenshot is only meant to demonstrate the feature for review, not for the readme. In fact multiple screenshots or gifs are encouraged. See #4046 and #4029